### ISSUE TITLE
Download of Backups via NCP Panel not working

### ISSUE THREAD

**Issue by hansaushamburg** [nextcloud/nextcloudpi#1252]
After the update to NC 20.0.6 downloading of backupfiles via the NCP panel is not working. Error message is "unauthorized access". Backupfiles are shown in the panel with a minus (-1356 MiB).

I checked via SSH. The backups are carried out properly. The restore function also works fine.

It's just the download function ...

-->

**System information**
NextCloudPi version  v1.34.7
NextCloudPi image    NextCloudPi_09-05-20
distribution         Raspbian GNU/Linux 10 \n \l
automount            yes
USB devices          sda sdb 
datadir              /media/myCloudDrive/ncdata
data in SD           no
data filesystem      btrfs
data disk usage      78G/3.7T
rootfs usage         2.9G/59G
swapfile             /var/swap
dbdir                /var/lib/mysql
Nextcloud check      ok
Nextcloud version    20.0.6.1
HTTPD service        up
PHP service          up
MariaDB service      up
Redis service        up
Postfix service      up
internet check       ok
port check 80        open
port check 443       open
IP                   ***REMOVED SENSITIVE VALUE***
gateway              ***REMOVED SENSITIVE VALUE***
interface            eth0
certificates         ***REMOVED SENSITIVE VALUE***
NAT loopback         no
uptime               23:07

```
</details>


<details>
<summary>Nextcloud configuration</summary>

```

{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "5": "nextcloudpi.local",
            "7": "nextcloudpi",
            "8": "nextcloudpi.lan",
            "11": "2a02:8108:483f:c0e4:fb82:e09c:6764:f91",
            "1": "192.168.178.61",
            "12": "cloud.hanslangen.de"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "20.0.6.1",
        "overwrite.cli.url": "https:\/\/cloud.hanslangen.de\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "tempdirectory": "\/media\/myCloudDrive\/ncdata\/tmp",
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "overwriteprotocol": "https",
        "maintenance": false,
        "logfile": "\/media\/myCloudDrive\/ncdata\/nextcloud.log",
        "loglevel": "2",
        "log_type": "file",
        "mail_sendmailmode": "smtp",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "ssl",
        "mail_smtpport": "465",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "theme": "",
        "filesystem_check_changes": 0,
        "data-fingerprint": "b193285538a8fa187a42b53f12080112",
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>


<details>
<summary>HTTPd logs</summary>

```

[Thu Jan 28 00:00:04.831539 2021] [ssl:warn] [pid 937:tid 3069805072] AH01909: localhost:4443:0 server certificate does NOT include an ID which matches the server name
[Thu Jan 28 00:00:04.832938 2021] [mpm_event:notice] [pid 937:tid 3069805072] AH00489: Apache/2.4.38 (Raspbian) OpenSSL/1.1.1d configured -- resuming normal operations
[Thu Jan 28 00:00:04.832968 2021] [core:notice] [pid 937:tid 3069805072] AH00094: Command line: '/usr/sbin/apache2'
[Thu Jan 28 13:06:15.430248 2021] [proxy_fcgi:error] [pid 16565:tid 2915243040] [client 192.168.178.32:49343] AH01071: Got error 'PHP message: PHP Fatal error:  Allowed memory size of 2147483647 bytes exhausted (tried to allocate 1608531968 bytes) in /var/www/ncp-web/download.php on line 68'
[Thu Jan 28 20:04:07.484759 2021] [proxy_fcgi:error] [pid 16569:tid 2898457632] [client 192.168.178.32:49358] AH01071: Got error 'PHP message: PHP Fatal error:  Allowed memory size of 2147483647 bytes exhausted (tried to allocate 1608531968 bytes) in /var/www/ncp-web/download.php on line 68'
[Thu Jan 28 20:18:28.059333 2021] [proxy_fcgi:error] [pid 16569:tid 2990777376] [client 192.168.178.32:49358] AH01067: Failed to read FastCGI header
[Thu Jan 28 20:18:28.059497 2021] [proxy_fcgi:error] [pid 16569:tid 2990777376] (104)Connection reset by peer: [client 192.168.178.32:49358] AH01075: Error dispatching request to :4443: 
[Thu Jan 28 20:31:23.739139 2021] [proxy_fcgi:error] [pid 16569:tid 2789319712] [client 192.168.178.32:49358] AH01071: Got error 'PHP message: PHP Fatal error:  Allowed memory size of 2147483647 bytes exhausted (tried to allocate 1608531968 bytes) in /var/www/ncp-web/download.php on line 68'
[Thu Jan 28 20:32:32.993434 2021] [proxy_fcgi:error] [pid 16565:tid 2856494112] [client 192.168.178.32:52599] AH01071: Got error 'PHP message: PHP Fatal error:  Allowed memory size of 2147483647 bytes exhausted (tried to allocate 1071661056 bytes) in /var/www/ncp-web/download.php on line 68'

```
</details>


<details>
<summary>Database logs</summary>

```

```
</details>


<details>
<summary>Nextcloud logs</summary>

```

{"reqId":"ZW18HkjjpwaULY3xf41m","level":1,"time":"2021-01-28T20:21:06+00:00","remoteAddr":"","user":"--","app":"updater","method":"","url":"--","message":"\\OC\\Updater::resetLogLevel: Reset log level to Warning(2)","userAgent":"--","version":"20.0.6.1"}
{"reqId":"YBMjDqD0naDWHNvks25JmgAAAFY","level":4,"time":"2021-01-28T20:48:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjLKD0naDWHNvks25JnAAAAFE","level":4,"time":"2021-01-28T20:48:45+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjSqD0naDWHNvks25JnQAAAE0","level":4,"time":"2021-01-28T20:49:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjaKD0naDWHNvks25JnwAAAFc","level":4,"time":"2021-01-28T20:49:45+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjhvMB4C301QPFWjmetQAAABg","level":4,"time":"2021-01-28T20:50:16+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjpKD0naDWHNvks25JoAAAAEk","level":4,"time":"2021-01-28T20:50:45+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMjwvMB4C301QPFWjmetgAAAA0","level":4,"time":"2021-01-28T20:51:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"webdav","method":"PROPFIND","url":"/remote.php/dav/files/Hans/","message":{"Exception":"Sabre\\DAV\\Exception\\ServiceUnavailable","Message":"Das System befindet sich im Wartungsmodus.","Code":0,"Trace":[{"file":"/var/www/nextcloud/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"checkMaintenanceMode","class":"OCA\\DAV\\Connector\\Sabre\\MaintenancePlugin","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":458,"function":"emit","class":"Sabre\\DAV\\Server","type":"->","args":["beforeMethod:PROPFIND",[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":251,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->","args":[{"__class__":"Sabre\\HTTP\\Request"},{"__class__":"Sabre\\HTTP\\Response"}]},{"file":"/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php","line":319,"function":"start","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/lib/Server.php","line":332,"function":"exec","class":"Sabre\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->","args":[]},{"file":"/var/www/nextcloud/remote.php","line":167,"args":["/var/www/nextcloud/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/var/www/nextcloud/apps/dav/lib/Connector/Sabre/MaintenancePlugin.php","Line":85,"CustomMessage":"--"},"userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmU6D0naDWHNvks25JygAAAE8","level":2,"time":"2021-01-28T21:02:12+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmU6D0naDWHNvks25JygAAAE8","level":2,"time":"2021-01-28T21:02:12+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmU6D0naDWHNvks25JygAAAE8","level":2,"time":"2021-01-28T21:02:12+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmVqD0naDWHNvks25JywAAAEY","level":2,"time":"2021-01-28T21:02:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmVqD0naDWHNvks25JywAAAEY","level":2,"time":"2021-01-28T21:02:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmVqD0naDWHNvks25JywAAAEY","level":2,"time":"2021-01-28T21:02:14+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmc-MB4C301QPFWjmezQAAABQ","level":2,"time":"2021-01-28T21:02:43+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmc-MB4C301QPFWjmezQAAABQ","level":2,"time":"2021-01-28T21:02:43+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmc-MB4C301QPFWjmezQAAABQ","level":2,"time":"2021-01-28T21:02:43+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/status.php","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmdfMB4C301QPFWjmezgAAABM","level":2,"time":"2021-01-28T21:02:46+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmdfMB4C301QPFWjmezgAAABM","level":2,"time":"2021-01-28T21:02:46+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}
{"reqId":"YBMmdfMB4C301QPFWjmezgAAABM","level":2,"time":"2021-01-28T21:02:46+00:00","remoteAddr":"31.17.236.67","user":"Hans","app":"no app in context","method":"GET","url":"/ocs/v2.php/core/navigation/apps?absolute=true&format=json","message":"Temporary directory /media/myCloudDrive/ncdata/tmp is not present or writable","userAgent":"Mozilla/5.0 (Macintosh) mirall/2.6.5legacy (build 20200710) (Nextcloud)","version":"20.0.6.1"}

```

**Comment by nachoparker** [comment]
Thanks,

This bug happens due to 32bit. We have 64bit images for the RPi now, but this has an easy workaround, so we'll fix.

**Comment by nachoparker** [comment]
fixed in `devel` v1.35.1, thanks
